src/components/DaysLists/index.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import Day from 'Components/Day';
+import PropTypes from 'prop-types';
+
+const DaysLists = ({ days }) => {
+  console.log(days);
+  return (
+    <>
+      {
+        days.map(({
+          date, projects, tracked_time,
+        }) => (
+          <Day
+            date={date}
+            projects={projects}
+            trackedTime={tracked_time}
+          />
+        ))
+      }
+    </>
+  );
+};
```

Comment:
Upss. Quedó un `console.log`, y también no hay necesidad del `return`, ya que la función puede hacer un return implicito.

```suggestion
const DaysLists = ({ days }) => (
    <>
      {
        days.map(({
          date, projects, tracked_time,
        }) => (
          <Day
            date={date}
            projects={projects}
            trackedTime={tracked_time}
          />
        ))
      }
    </>
  );
```